.github/workflows/wheels.yml

```diff
@@ -50,10 +50,12 @@ jobs:
       run: |
         python -m cibuildwheel --output-dir wheelhouse
       env:
-        CIBW_SKIP: "pp* cp27-* cp34-* cp35-* *i686*"
+        CIBW_SKIP: "pp* cp310-* *i686*"
```

Comment:
curious why you're skipping 3.10?

Reply:
Only because it caused a failure (https://github.com/tomwhite/cyvcf2/runs/3463959682?check_suite_focus=true) - not sure that is related to this PR though since the build hasn't run for a while.